plugins.yaml

```diff
@@ -436,3 +436,10 @@ aiida-zeopp:
   entry_point_prefix: zeopp
   pip_url: aiida-zeopp
   plugin_info: https://raw.github.com/ltalirz/aiida-zeopp/master/setup.json
+aiida-qp2:
+  code_home: https://github.com/TREX-CoE/aiida-qp2
+  development_status: beta
```

Comment:
We've deprecated this variable and suggest you use the standard PyPI trove classifiers in the plugin (e.g. in the setup.json)
```suggestion
```

Reply:
Thank you @ltalirz ! Good point, I used the trove classifier but decided to duplicate it here. I will remove it.